pkg/controller/common.go

```diff
@@ -305,6 +308,64 @@ func (c *SharedState) deleteOperationQueue(volumeName string) {
 	lockable.Unlock()
 }
 
+func (c *SharedState) createReplicaRequestsQueue() {
+	heap.Init(&c.priorityReplicaRequestsQueue.queue)
+	heap.Init(&c.failedReplicaRequestsQueue.queue)
+}
+
+func (c *SharedState) generateReplicaAttachmentRoutines(ctx context.Context, azr azReconciler, namespace string) error {
```

Comment:
While technically correct that `generateReplicaAttachmentRoutines` creates `goroutine`, this is an implementation detail of processing the pending replicas. Consider `processPendingReplicaQueues` instead.